modules/admin-top-bar/assets/scss/admin.scss

```
@@ -124,9 +124,11 @@ $top-bar-height: 50px;
 		}
 	}
 
-	&:not(.top-bar-active ) ~ #wpbody {
-		.wrap h1 {
-			display: block;
+	&.e-admin-top-bar--not-active ~ #wpbody {
+		.wrap {
+			h1, .page-title-action {
+				display: inline-block;
+			}
 		}
```

Maybe if you want:

```suggestion
		.wrap :where( h1, .page-title-action ) {
			display: inline-block;
		}
```

Just coz I like new CSS :D

---

The linter yells it doesn't recognize the pseudo. I leave it as is.